internal/controller/api_to_config.go

```diff
@@ -146,9 +146,21 @@ func routerToFRRConfig(r v1beta1.Router, alwaysBlock []frr.IncomingFilter, secre
 }
 
 func neighborToFRR(n v1beta1.Neighbor, prefixesInRouter []string, alwaysBlock []frr.IncomingFilter, routerVRF string, passwordSecrets map[string]corev1.Secret, bfdProfiles map[string]*frr.BFDProfile) (*frr.NeighborConfig, error) {
-	neighborFamily, err := ipfamily.ForAddresses(n.Address)
-	if err != nil {
-		return nil, fmt.Errorf("failed to find ipfamily for %s, %w", n.Address, err)
+	if n.Address == "" && n.Interface == "" {
+		return nil, fmt.Errorf("neighbor %s has no Address or Interface specified", neighborName(n))
```

Comment:
small nit: I think the `neighborName` would be unclear here, we can just say something about "neighbor must have an address or interface specified"

Reply:
I am not sure I follow you,
Now the error is
```
failed to process neighbor 65010@ for router 65010-: neighbor 65010@ has no Address or Interface specified
```
the `neighborName` is a bit better the nothing, no?

Reply:
`neighbor 65010@` is ambiguous, I think it is better to just say "there is a neighbor with no interface and address"

Reply:
done